fftools/ffmpeg_hw.c

```diff
@@ -527,14 +527,12 @@ int hw_device_setup_for_filter(FilterGraph *fg)
     HWDevice *dev;
     int i;
 
-    // If the user has supplied exactly one hardware device then just
-    // give it straight to every filter for convenience.  If more than
-    // one device is available then the user needs to pick one explcitly
-    // with the filter_hw_device option.
+    // Pick the last hardware device if the user doesn't pick the device for
+    // filters explcitly with the filter_hw_device option.
```

Comment:
typo: explcitly -> explicitly

Reply:
Thanks for catching the typo. I will update it.